Tests do not compile on Android (Termux)

The following error occurs during compilation:
```
~/xcmdparser $ cmake --build build
[ 15%] Built target xcmdparser
[ 23%] Building C object tests/CMakeFiles/xcmdparser_test.dir/argument_test.c.o
In file included from /data/data/com.termux/files/home/xcmdparser/tests/argument_test.c:1:
In file included from /data/data/com.termux/files/home/xcmdparser/tests/common.h:3:
/data/data/com.termux/files/home/xcmdparser/tests/utest.h:320:5: error: call to undeclared function 'timespec_get'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
    timespec_get(&ts, TIME_UTC);
    ^
1 error generated.
make[2]: *** [tests/CMakeFiles/xcmdparser_test.dir/build.make:76: tests/CMakeFiles/xcmdparser_test.dir/argument_test.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:898: tests/CMakeFiles/xcmdparser_test.dir/all] Error 2
make: *** [Makefile:101: all] Error 2
```

--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -1,5 +1,7 @@
-file(GLOB_RECURSE XCMDPARSER_TEST_SRC "*.c")
+if(NOT ${CMAKE_SYSTEM_NAME} MATCHES "Android")
+    file(GLOB_RECURSE XCMDPARSER_TEST_SRC "*.c")
 
-add_executable(xcmdparser_test ${XCMDPARSER_TEST_SRC} "common.h")
+    add_executable(xcmdparser_test ${XCMDPARSER_TEST_SRC} "common.h")
 
-target_link_libraries(xcmdparser_test xcmdparser)
+    target_link_libraries(xcmdparser_test xcmdparser)
+endif()

